coral/src/app/features/approvals/schemas/components/SchemaApprovalsTable.tsx

```diff
@@ -76,6 +82,17 @@ function SchemaApprovalsTable({
         };
       },
     },
+    {
+      type: "custom",
+      field: "forceRegister",
+      headerName: "Force register",
+      UNSAFE_render: ({ forceRegister }: { forceRegister: boolean }) => {
+        if (forceRegister) {
+          return <StatusChip text={"Yes"} status={"danger"} dense={true} />;
```

Comment:
I imagine there has been some conversation about how to design this, but maybe the column could be called "Registration", and then we could have a red pill "Forced" and a green pill "Standard". I feel it would be a little bit more expressive than a red "Yes" 🤔

Reply:
I like that idea! I'll check with Venla (somehow I can't tag here here, even so I can on issues) 🏃

Reply:
@mathieu-anderson  Idea is good. I do think that it might give the impression that you are doing a "Standard force registration".

Maybe the top column could be more generic like "Additional information/notes/lables" -> if force register applies chip with "Force register applied".